cassandra/src/main/java/io/micronaut/configuration/cassandra/CassandraConfiguration.java

```diff
@@ -26,21 +25,25 @@
  * used to denote the primary cluster bean.
  *
  * @author Nirav Assar
+ * @author Michael Pollind
  * @since 1.0
  */
-@EachProperty(value = "cassandra", primary = "default")
+@EachProperty(value = CassandraConfiguration.PREFIX, primary = "default")
 public class CassandraConfiguration {
+    public static final String PREFIX = "cassandra.datasource";
```

Comment:
Is this a value you've chosen, or one dictated by the new driver?

Reply:
It's a value that I choose but we can use something better if that is more appropriate. I was looking at the datastax document and I was thinking of maybe adding cassandra.dao, cassandra.mapping etc ...